Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml

```
@@ -54,6 +61,11 @@
   - !type:AddComponentSpecial
     components:
       - type: CommandStaff
+  alternateTitles:
+  - HoPTitleFirstMate
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)